staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -1283,28 +1271,6 @@ func exportObjectMeta(accessor metav1.Object, exact bool) {
 	}
 }
 
-// Export implements the rest.Exporter interface
-func (e *Store) Export(ctx context.Context, name string, opts metav1.ExportOptions) (runtime.Object, error) {
-	obj, err := e.Get(ctx, name, &metav1.GetOptions{})
-	if err != nil {
-		return nil, err
-	}
-	if accessor, err := meta.Accessor(obj); err == nil {
-		exportObjectMeta(accessor, opts.Exact)
-	} else {
-		klog.V(4).Infof("Object of type %v does not have ObjectMeta: %v", reflect.TypeOf(obj), err)
-	}
-
-	if e.ExportStrategy != nil {
-		if err = e.ExportStrategy.Export(ctx, obj, opts.Exact); err != nil {
-			return nil, err
-		}
-	} else {
-		e.CreateStrategy.PrepareForCreate(ctx, obj)
```

Comment:
...? so strange...

Reply:
> ...? so strange...

Very. I think removing it is proper, but yeah, that's weird.